### ISSUE TITLE
Blank page on install

### ISSUE THREAD

**Issue by abeborker** [ljvillanueva/pumilio#64]
Hi Luis,

I'm trying to install pumilio for my first time on ubuntu.  I got all the green checkboxes in the install page, but then I click install and am greeted with a white page at http://localhost/pumilio/

I checked the database and the installer did create a slew of tables.

Any suggestions on troubleshooting?

Thanks for this great software, I'm looking forward to using it!

Cheers,
Abe

**Comment by ljvillanueva** [comment]
Which version were you installing?

Can you check the contents of the apache log? That should indicate if there is a missing program or configuration issue.

**Comment by abeborker** [comment]
Thanks for the help!

There are indeed some errors in there.  Any suggestions?

```
[Tue Mar 22 15:13:33.730310 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined index: HTTPS in /var/www/html/pumilio/include/apply_config.php on line 9
[Tue Mar 22 15:13:34.937366 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined variable: app_custom_name in /var/www/html/pumilio/include/apply_config.php on line 79
[Tue Mar 22 15:13:34.937449 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined variable: use_googlemaps in /var/www/html/pumilio/include/apply_config.php on line 89
[Tue Mar 22 15:13:34.937537 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 121
[Tue Mar 22 15:13:34.937580 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 124
[Tue Mar 22 15:13:34.962189 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Notice:  Undefined variable: default_qf in /var/www/html/pumilio/index.php on line 27
[Tue Mar 22 15:13:35.097129 2016] [:error] [pid 19955] [client 128.114.44.136:54056] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Sounds.SoundStatus' in 'where clause'' in /var/www/html/pumilio/libs/DB/DB.php:94\nStack trace:\n#0 /var/www/html/pumilio/libs/DB/DB.php(94): PDOStatement->execute(NULL)\n#1 /var/www/html/pumilio/libs/DB/DB.php(75): DByte\\DB::query('SELECT `Sites`....', NULL)\n#2 /var/www/html/pumilio/index.php(47): DByte\\DB::fetch('SELECT `Sites`....')\n#3 {main}\n  thrown in /var/www/html/pumilio/libs/DB/DB.php on line 94
[Tue Mar 22 17:19:26.712891 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined index: HTTPS in /var/www/html/pumilio/include/apply_config.php on line 9
[Tue Mar 22 17:19:26.856454 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined variable: app_custom_name in /var/www/html/pumilio/include/apply_config.php on line 79
[Tue Mar 22 17:19:26.856518 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined variable: use_googlemaps in /var/www/html/pumilio/include/apply_config.php on line 89
[Tue Mar 22 17:19:26.856533 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 121
[Tue Mar 22 17:19:26.856542 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 124
[Tue Mar 22 17:19:26.865889 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Notice:  Undefined variable: default_qf in /var/www/html/pumilio/index.php on line 27
[Tue Mar 22 17:19:26.876467 2016] [:error] [pid 19958] [client 127.0.0.1:43622] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Sounds.SoundStatus' in 'where clause'' in /var/www/html/pumilio/libs/DB/DB.php:94\nStack trace:\n#0 /var/www/html/pumilio/libs/DB/DB.php(94): PDOStatement->execute(NULL)\n#1 /var/www/html/pumilio/libs/DB/DB.php(75): DByte\\DB::query('SELECT `Sites`....', NULL)\n#2 /var/www/html/pumilio/index.php(47): DByte\\DB::fetch('SELECT `Sites`....')\n#3 {main}\n  thrown in /var/www/html/pumilio/libs/DB/DB.php on line 94
```

**Comment by ljvillanueva** [comment]
Try going to http://localhost/pumilio/upgrade
and try again.

If it doesn't work, which version did you install?

**Comment by abeborker** [comment]
*** After writing this, I did get to the login screen, read at the end

When I goto /upgrade/ I get this:
This system has a database that is too old or was edited manually.

I made the sql database in phpmyadmin, although it is throwing up this error:  Connection for controluser as defined in your configuration failed.

I'm installing 2.7.6 I downloaded yesterday.  Sorry to be a bother, thank you for the help!

Server: Localhost via UNIX socket
Server type: MySQL
Server version: 5.5.47-0ubuntu0.14.04.1 - (Ubuntu)
Protocol version: 10
Server charset: UTF-8 Unicode (utf8)
Web server

Apache/2.4.7 (Ubuntu)
Database client version: libmysql - 5.5.47
PHP extension: mysqli Documentation

****Follow up:  Just for the heck of it I tried reinstalling, recreating the database a few times, and by doing the right sequence, I have the login page, but the login credentials I entered on the install page.  this the error from the apache log:

Also FWIW the trick to getting the install to take was starting with a freshly created db, removing config.php, run install, then create config.php.  Also the install page returns a blank white page, not a confirmation or anything else.

ok, the error log:
`[Tue Mar 22 18:36:30.929540 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined index: HTTPS in /var/www/html/pumilio/include/apply_config.php on line 9, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.932209 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: app_custom_name in /var/www/html/pumilio/include/apply_config.php on line 79, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.932290 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: use_googlemaps in /var/www/html/pumilio/include/apply_config.php on line 89, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.932318 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 121, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.932338 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 124, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.932813 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: where_to in /var/www/html/pumilio/login.php on line 89, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.939133 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined index: HTTPS in /var/www/html/pumilio/include/apply_config.php on line 9, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.940017 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: app_custom_name in /var/www/html/pumilio/include/apply_config.php on line 79, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.940052 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: use_googlemaps in /var/www/html/pumilio/include/apply_config.php on line 89, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.940062 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 121, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.940069 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: mapping_system in /var/www/html/pumilio/include/apply_config.php on line 124, referer: http://localhost/pumilio/login.php?e=1
[Tue Mar 22 18:36:30.940444 2016] [:error] [pid 1526] [client 127.0.0.1:44081] PHP Notice:  Undefined variable: use_leaflet in /var/www/html/pumilio/login.php on line 168, referer: http://localhost/pumilio/login.php?e=1
find: `tmp/*': No such file or director`

**Comment by abeborker** [comment]
Ah I take it back, I downloaded the zip off github and apparrently am trying to install 3.0?

from the about page:
Powered by Pumilio
Pumilio v. 3.0.0.dev - A sound archive manager and visualization web application. http://ljvillanueva.github.io/pumilio 
Copyright (©) 2010-2016 Luis J. Villanueva-Rivera (ljvillanueva@coquipr.com)

maybe that explains things?

**Comment by abeborker** [comment]
Yup, all fixed now that I found 2.76 

Sorry for all the bother, looking forward to v3 eventually!

Cheers,
Abe
